src/globus_cli/termio/errors.py

```diff
@@ -13,17 +15,17 @@ class PrintableErrorField:
 
     TEXT_PREFIX = "Globus CLI Error:"
 
-    def __init__(self, name, value, multiline=False):
+    def __init__(self, name: str, value: str | int | None, multiline: bool = False):
```

Comment:
I thought that mypy required that even `__init__()` methods needed a trailing `-> None` annotation. Maybe that was a past version, or just an incorrect memory...

Reply:
No, you're correct! That's definitely a requirement of strict mode. Let me add it really quickly. I'm sure something in our settings (which are not so simple as `--strict`) is interfering, so we'd just have to come back to this later if I don't.

Reply:
I would have thought that `disallow-incomplete-defs = true` would catch this... Maybe a bug triggered by our config?
It's fixed now anyway.